data/guardian.go

```diff
@@ -0,0 +1,23 @@
+package data
+
+// GuardianDataResponse holds the guardian data endpoint response
+type GuardianDataResponse struct {
+	Data struct {
+		GuardianData *GuardianData `json:"guardianData"`
+	} `json:"data"`
+	Error string `json:"error"`
+	Code  string `json:"code"`
+}
+
+// Guardian holds the relevant information for an account guardian
+type Guardian struct {
```

Comment:
can you import the GuardianData and Guardian structures from elrond-go-core instead?